public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -284,3 +285,13 @@ export function isQueryWithLineFilter(query: string): boolean {
 
   return queryWithLineFilter;
 }
+
+export function getStreamSelectorsFromQuery(query: LokiQuery): string[] {
```

Comment:
I'd suggest to have here only query string, to follow the same pattern as we use for other utils function. We only need query string anyway. 🙂